app/src/main/java/org/simple/clinic/summary/prescribeddrugs/DrugSummaryModel.kt

```diff
@@ -2,14 +2,23 @@ package org.simple.clinic.summary.prescribeddrugs
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.drugs.PrescribedDrug
 import java.util.UUID
 
 @Parcelize
 data class DrugSummaryModel(
-    val patientUuid: UUID
+    val patientUuid: UUID,
+    val prescribedDrugs: List<PrescribedDrug>?
 ) : Parcelable {
 
   companion object {
-    fun create(patientUuid: UUID) = DrugSummaryModel(patientUuid)
+    fun create(patientUuid: UUID) = DrugSummaryModel(
+        patientUuid = patientUuid,
+        prescribedDrugs = null
```

Comment:
We won't be passing it using the factory constructor. That's why I haven't added it to `create`. So basically this value will be `null` before the repository call is made, after that it will be replaced with the data from the db.